rero_ils/modules/acq_order_lines/api.py

```diff
@@ -279,6 +282,30 @@ def get_note(self, note_type):
         ]
         return next(iter(note), None)
 
+    def get_links_to_me(self, get_pids=False):
+        """Record links.
+
+        :param get_pids: if True list of linked pids
+                         if False count of linked records
+        """
+        links = {}
+        query = AcqReceiptLinesSearch()\
+            .filter('term', acq_order_line__pid=self.pid)
+
+        receipt_lines = sorted_pids(query) if get_pids else query.count()
+
+        if receipt_lines:
+            links['acq_receipt_lines'] = receipt_lines
+
+        return links
+
+    def reasons_not_to_delete(self):
+        """Get reasons not to delete record."""
+        cannot_delete = {}
+        if links := self.get_links_to_me():
+            cannot_delete['links'] = links
+        return cannot_delete
+
```

Comment:
I don't understand why you have this code in your PR.
See https://github.com/rero/rero-ils/pull/2986

Reply:
A rebase was needed. This is the case now.